WikiCohereEnglishEmbeddingOnlyScenario.cs

```diff
@@ -1,9 +1,24 @@
-﻿using Microsoft.Extensions.Configuration;
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.Configuration;
 namespace VectorIndexScenarioSuite
 { 
-    internal class WikiCohereEnglishEmbeddingOnlyScenario : WikiCohereEmbeddingOnlyBaseSceario
+    internal class WikiCohereEnglishEmbeddingOnlyScenario : EmbeddingOnlyScearioBase
     {
-        private const string GROUND_TRUTH_FILE = "wikipedia_truth";
+        protected override string BaseDataFile => "wikipedia_base";

```

Comment:
can we keep the base for wikiCohere and move duplicate stuff from streaming and non-streaming there please?